src/shared/format-table.c

```diff
@@ -751,6 +767,16 @@ int table_add_many_internal(Table *t, TableDataType first_type, ...) {
                         data = &buffer.int_val;
                         break;
 
+                case TABLE_INT8:
+                        buffer.int8 = va_arg(ap, int);
```

Comment:
not that it matters, but maybe:

```c
case TABLE_INT8: {
        int x = va_arg(ap, int);
        assert(x >= UINT8_MIN && x <= UINT8_MAX);

        buffer.int8 = x;
        …
```

i.e. catch some usage errors, given that we can for free?